test/components/activity/upload_table.spec.jsx

```diff
@@ -72,6 +69,6 @@ describe('UploadTable', () => {
     );
 
     const rowElements = TestTable.querySelectorAll('tr.upload');
-    expect(rowElements.length).to.eq(2);
+    expect(rowElements.length).to.eq(0);
```

Comment:
This change means it's no longer testing what it is intended to test, which is that UploadTable will render elements for the uploads that are passed in as props. It's doing that by looking for `tr` nodes with the class `upload`, but since it's not longer rendered as a table with rows, you should change the selector to look for the new structure. Maybe that's just `div.upload` instead of `tr.upload`.